logging/src/test/resources/default-subsystem.xml

```diff
@@ -38,9 +38,6 @@
     <logger category="com.arjuna">
         <level name="WARN"/>
     </logger>
-    <logger category="org.apache.tomcat.util.modeler">
-        <level name="WARN"/>
-    </logger>
```

Comment:
@jamezp WDYT about this one? The rest of them either affect testsuite configs or only affect servers using a current installation, where jboss-web is not used. By this one will affect the logging system in our standard domain.xml profiles, and those profiles can be adapted for use with legacy slaves that still have jboss-web.

OTOH, those profiles would have to be *adapted* for use; i.e. the user isn't just taking our standard profile and using it. So I think having to notice and deal with these kinds of subtle behavior differences isn't unreasonable, and this patch is ok.